dask_cuda/benchmarks/utils.py

```diff
@@ -176,8 +183,11 @@ def get_scheduler_workers(dask_scheduler=None):
 
 
 def setup_memory_pool(pool_size=None, disable_pool=False):
-    import rmm
-    import cupy
+    try:
+        import rmm
+        import cupy
+    except ImportError:
+        return None
```

Comment:
I think it doesn't make sense to just `return None` if RMM isn't installed, I hate this kind of silent fallbacks because it makes it very hard to debug. IMO, we should error out if RMM can't be imported and `pool_size != None or disable_pool is not False`.

Reply:
That's a good point.  My goal was minimal changes in the merge code .  Would you be ok if I added in some more gpu/cpu conditionals  ?